test/specs/trailApexReplayDebugger.e2e.ts

```diff
@@ -0,0 +1,228 @@
+    expect(successNotificationWasFound).toBe(true);
+
+    // Verify test results are listed on vscode's Output section
+    const outputPanelText = await utilities.attemptToFindOutputPanelText('Apex', '=== Test Results', 10);
+    expect(outputPanelText).not.toBeUndefined();
+    expect(outputPanelText).toContain('Assertion Failed: incorrect ticker symbol');
+    expect(outputPanelText).toContain('Expected: CRM, Actual: SFDC');
+  });
+
+  step('SFDX: Get Apex Debug Logs', async () => {
+    // Run SFDX: Get Apex Debug Logs
+    const workbench = await browser.getWorkbench();
+    prompt = await utilities.runCommandFromCommandPrompt(workbench, 'SFDX: Get Apex Debug Logs', 1);
+
+    // Wait for the command to execute
+    await utilities.waitForNotificationToGoAway(workbench, 'Getting Apex debug logs', fiveMinutes);
+
+    // Select a log file
+    const quickPicks = await prompt.getQuickPicks();
+    expect(quickPicks).not.toBeUndefined();
+    expect(quickPicks.length).toBeGreaterThanOrEqual(1);
+    await prompt.selectQuickPick('User User - ApexTestHandler');
+
+    // Wait for the command to execute
+    await utilities.waitForNotificationToGoAway(workbench, 'Running SFDX: Get Apex Debug Logs', fiveMinutes);
+
+    const successNotificationWasFound = await utilities.notificationIsPresent(workbench, 'SFDX: Get Apex Debug Logs successfully ran');
+    expect(successNotificationWasFound).toBe(true);
+
+    // Verify content on vscode's Output section
+    const outputPanelText = await utilities.attemptToFindOutputPanelText('Apex', 'Starting SFDX: Get Apex Debug Logs', 10);
+    expect(outputPanelText).not.toBeUndefined();
+    expect(outputPanelText).toContain('|EXECUTION_STARTED');
+    expect(outputPanelText).toContain('|EXECUTION_FINISHED');
+    expect(outputPanelText).toContain('ended SFDX: Get Apex Debug Logs');
+
+    // Verify content on log file
+    const editorView = workbench.getEditorView();
+    const activeTab = await editorView.getActiveTab();
+    const title = await activeTab?.getTitle();
+    const textEditor = await editorView.openEditor(title!) as TextEditor;
+    const executionStarted = await textEditor.getLineOfText('|EXECUTION_STARTED');
+    const executionFinished = await textEditor.getLineOfText('|EXECUTION_FINISHED');
+    expect(executionStarted).toBeGreaterThanOrEqual(1);
+    expect(executionFinished).toBeGreaterThanOrEqual(1);
+  });
+
+  step('Replay an Apex Debug Log', async () => {
+    // Run SFDX: Launch Apex Replay Debugger with Current File
+    const workbench = await browser.getWorkbench();
+    await utilities.runCommandFromCommandPrompt(workbench, 'SFDX: Launch Apex Replay Debugger with Current File', 5);
+
+    // Continue with the debug session
+    await browser.keys(['F5']);
+    await utilities.pause(1);
+    await browser.keys(['F5']);
+    await utilities.pause(1);
+  });
```

Comment:
After the debugging finishes (line 181), I think the following should be checked:

switch to Debug Console
in the Debug console, look for `"launchRequest: args={"name":"Launch Apex Replay Debugger","type":"apex-replay","request":"launch","`
also look for `|EXECUTION_STARTED`
also look for `|EXECUTION_FINISHED`
also look for `Apex Replay Debugger session terminated.`

Reply:
For getting the text of the debug console tab, as you've seen `debugConsoleView.getText()` doesn't work.  You will need to implement something like `getOutputViewText()` (in test/utilities/outputView.ts) but for the debug console.

I suggest creating a new file, /utilities/debugConsolView.ts, and add a `getDebugConsoleViewText()` function.

Reply:
Darn.  Well, after discussing with Cristi, it turns out that:
* one can't call debugConsoleView.getText() (due to a bug in wdio)
* and the debug console doesn't support select all like the output panel does, so we can't implement a solution like the one that's in OutputView